rasa/shared/core/domain.py

```diff
@@ -154,6 +156,7 @@ def from_yaml(cls, yaml: Text, original_filename: Text = "") -> "Domain":
                 yaml, rasa.shared.constants.DOMAIN_SCHEMA_FILE
             )
         except rasa.shared.utils.validation.InvalidYamlFileError as e:
+            e.filename = original_filename
```

Comment:
what do you mean? the `InvalidYamlFileError` exception handles this when converting itself to a string (it will not add a filename if it is not set or set to "")

Reply:
ok gotcha, so let's keep it 👍